pages/workspaces/[workspaceId]/contributors.tsx

```diff
@@ -68,6 +68,7 @@ export default function WorkspaceContributorsPage({ workspace }: WorkspaceContri
           username: info.contributor.username,
           updated_at: info.contributor.updated_at,
           user_id: info.contributor_id,
+          devstats_updated_at: info.contributor.updated_at,
```

Comment:
Is this supposed to be `info.contributor.devstats_updated_at`?

Reply:
oops, yep. Will fix.

Reply:
I also have a PR up for the API that will make this PR work.

Reply:
Fixed @brandonroberts!